problemsets-mat223/linearalbegra.tex

```diff
@@ -2647,18 +2647,50 @@ \section*{Change of Basis \& Coordinates}
 				\[
 					\vec v=2\vec e_1+2\vec e_2
 					=2(3\vec c_1-\vec c_2)+2(-5\vec c_1+2\vec c_2)
-					=-4c_1+2c_2.
+					=-4\vec c_1+2\vec c_2.
 				\]
 			\end{solution}
 		\item Can you find a matrix $X$ so that $X[\vec w]_{\mathcal C} = [\vec w]_{\mathcal E}$
 			for any	$\vec w$?
-			\begin{solution}[inline]
-				$X = \mat{2&5\\1&3}$ is such a matrix. 
+			\begin{solution}
+				$X = \mat{2&5\\1&3}$ is such a matrix.
+
+				For a vector $\vec w$, if $[\vec w]_{\mathcal C}=\mat{x\\y}$ then
```

Comment:
I would like this proof to go as follows: 

$X$ is 2x2, so $X=[a b; c d]$ for some a,b,c,d\in R. We know [..]_C=[..]_E and so X[..]_C=[..]_E. Similarly, [..']_C=[..']_E
and so X[..']_C=[..']_E. We can now solve for a,b,c,d to get...

Reply:
Ah. Do you mean you want those relationships to be between the basis vectors? I mean something like:

X=[a b; c d] for some a,b,c,d\in R, and we know 

[1 0]_C = [2 1]_E and [0 1]_C = [5 3]_E. 

So we need 

X[1 0]_C = [a c] = [2 1]_E and X[0 1]_C = [b d] = [5 3]_E

Reply:
That's what I mean!